fluid-project/hearth
# feat: add controller, routes, and views for ResourceCollection model (resolves #142)

Adds controller, routes, and views for ResourceCollection model. Also, there's a new composer package, laravel-unique-translation: [https://github.com/codezero-be/laravel-unique-translation](url) used for checking translated value is unique.

**Testing**

- Verify that authorized user can create resource collections
- Verify that authorized user can modify resource collections belong to them
- Verify that authorized user can delete resource collections belong to them, with correct password
- Verify validation of create resource collection request
- Verify validation of update resource collection request
